### PR TITLE
More customization

### PR DESCRIPTION
**Pull request by MegaVasiliy007**
- Fix problem with outputBucket
- Add outputKey config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![Build](https://github.com/deek87/lamba-thumbnailer/workflows/Build/badge.svg?branch=master)
+![Build](https://github.com/MegaVasiliy007/lamba-thumbnailer/workflows/Build/badge.svg?branch=master)
```
Comment (deek87) [inline README.md]: no need for this

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "lambda-video-thumbnailer",
-  "version": "2.0.0-alpha",
+  "name": "lambda-video-thumbnailer2",
```
Comment (deek87) [inline package.json]: i dont know why you are changing the version and package name

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+  "version": "1.0.0",
   "description": "A Package that allows users to automatically create thumbnails when uploaded to s3.",
-  "homepage": "https://github.com/deek87/lamba-thumbnailer#readme",
+  "homepage": "https://github.com/MegaVasiliy007/lamba-thumbnailer#readme",
```
Comment (deek87) [inline package.json]: again changing repository

**Review by deek87 — CHANGES_REQUESTED** [review]
I have no problem adding this stuff, for when i get to round to finishing this version
but you need to remove all the version/repository changes.
If not I'll just cherry pick them

File: package.json (hunk 2 above)
Comment (MegaVasiliy007) [reply package.json]: I needed changes in the published package, and there was no response to the pull request for a long time.

**Comment by deek87** [comment]
no problem, i didnt see the notification for this PR sorry about that

**Pull request merged**